### ISSUE TITLE
Warning: React does not recognize the `checkOptions` prop on a DOM element

### ISSUE THREAD

**Issue by asmakhalfallah2018** [sstur/react-rte#377]
Hello , i started working with react-rte today . It's an amazing library and it matches my requirements but i am wondering why this warning is showing up . 

`Warning: React does not recognize the `checkOptions` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `checkoptions` instead. If you accidentally passed it from a parent component, remove it from the DOM element.
    in button (created by e)
    in e (created by e)
    in div (created by s)
    in s (created by e)
    in e (created by e)
    in e (created by e)
    in div (created by c)
    in c (created by e)
    in div (created by e)
    in e (created by e)
    in div (created by e)
    in e (created by MyStatefulEditor)
    in div (created by MyStatefulEditor)
    in MyStatefulEditor (created by Main)
    in div (created by Main)
    in Main (created by Main)
    in div (created by GraphObjPage)
    in GraphObjPage (created by Main)
    in div (created by Main)
    in Main (created by main)
    in div (created by CustomView)
    in CustomView (created by main)
    in div (created by main)
    in main (created by Context.Consumer)
    in Route
    in div
    in Router (created by HashRouter)
    in HashRouter`

@sstur can you provide us a new version (including the last fix of checkOptions) please ?

**Comment by asmakhalfallah2018** [comment]
@sstur can you provide us a new version (including the last fix of  checkOptions) please ?

**Comment by rpilev** [comment]
Same issue here.

**Comment by myoneflag** [comment]
Same issue here. Hope new version.

**Comment by machineghost** [comment]
We have a fix for this already merged in: https://github.com/sstur/react-rte/pull/376#event-3681541994

All the work has already been completed: could we please get a release?

**Comment by machineghost** [comment]
I'm not sure what's holding up the release of this (it was literally a one-line change), but in the meantime if anyone wants to get rid of that giant wall of red text in their console, you can use this temporary fix:

    "react-rte": "https://github.com/machineghost/react-rte/",

Make sure you `npm i` after the change.  However, I should note that my repo won't get future updates, so this is only a temporary workaround until the fix is released properly.

**Comment by asmakhalfallah2018** [comment]
> I'm not sure what's holding up the release of this (it was literally a one-line change), but in the meantime if anyone wants to get rid of that giant wall of red text in their console, you can use this temporary fix:
> 
> ```
> "react-rte": "https://github.com/machineghost/react-rte/",
> ```
> 
> Make sure you `npm i` after the change. However, I should note that my repo won't get future updates, so this is only a temporary workaround until the fix is released properly.

**Comment by myoneflag** [comment]
When the plan to release a new version?

**Comment by jgbright** [comment]
The pull request that fixes this has been merged into master, but a new release hasn't been produced yet. 
 For my project, I just installed the latest commit on master as of today.  My build is still deterministic (doesn't follow a branch), but I need to remember to go back to the regular react-rte package when it's available.

```
npm install --save git+https://github.com/sstur/react-rte/#8c81622706a5f8856d39497bf33f92a97e9664fc
```

**Comment by webmobiles** [comment]
please tell me when a new version is possible.

**Comment by MForMarlon** [comment]
This and #374 are the same issue.

**Comment by duhaime** [comment]
@sstur any chance we could get a release? 

It would be great to clear this error from the console.

**Comment by sstur** [comment]
Done. This was released with [v0.16.4](https://github.com/sstur/react-rte/releases/tag/v0.16.4)
